modules/ding_react/js/ding-react.reset.js

```diff
@@ -0,0 +1,3 @@
+// For some reason resetting will not work if run to early. Perhaps the
+// persisted store is not quite ready. Consequently we have to wait a bit.
+setTimeout(() => window.ddbReact.reset(), 500);
```

Comment:
Will this not assume at all uses have "fast" browsers? What on a slow/older mobil? Could this be solved better in react in ddbReact. So it makes the user logged out checks and resets it self. I don't know the react code... so just making suggestions.

Reply:
I am not sure this is a problem. 

The comments on rt2zz/redux-persist#1015 do not mention anything about device performance and generally refer to 200 ms timeout. This has 500 ms - to be sure I guess.

In general we need some kind of signal to know when to run our code. This has to be in relation to Drupal. The React components by design do not know anything about the underlying application layer anyway.

As I see it we have two options:

1. Hijack the logout button and run `reset()` when the user clicks the button but before the serverside logout logic is called. This means that the React code will have to be added to every page compared to the current situation where it is only loaded on pages with actual components.
2. Run our code on the logout page as soon as it loads.

This PR chooses option #2 in the current form.